server/src/main/java/org/spine3/server/entity/VisibleEntity.java

```diff
@@ -18,27 +18,24 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package org.spine3.change;
+package org.spine3.server.entity;
 
-import static com.google.common.base.Preconditions.checkArgument;
-import static com.google.common.base.Preconditions.checkNotNull;
+import com.google.protobuf.Message;
 
 /**
- * Routines common for all mismatches.
+ * An entity which has visibility.
```

Comment:
Can we reference some definition of `visibility`? Or something that would allow readers to catch up on the subject.